.eslintrc.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    "env": {
+        "browser": true
+    },
+    "extends": "eslint:recommended",
+    "rules": {
+        "no-console": [ "off" ],
+        "indent": [ "error", 4 ],
+        "linebreak-style": [ "error", "windows" ],
```

Comment:
Looks like the original version was developed on windows. Should we update the line endings?